api/candlepin-api-spec.yaml

```diff
@@ -8435,7 +8435,7 @@ components:
               owner:
                 $ref: '#/components/schemas/NestedOwnerDTO'
               environment:
-                $ref: '#/components/schemas/EnvironmentDTO'
+                type: string
```

Comment:
This is not supposed to be a String, but an EnvironmentDTO, whose only field being populated will be the 'name' field, which will be a comma-separated list of all the environment names

Reply:
ah I thought It's only EnvironmentDTO’s ‘name’ field.
Fixed it.